### PR TITLE
[data]: added mrabro profile

### PR DESCRIPTION
**Pull request by mrabro**
- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

Added my profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2330"></a>

### CODE REVIEW

File: data/mrabro.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Rafi Abro",
+  "type": "personal",
+  "bio": "I'm Rafi, a full-time full-stack developer 👨‍💻 working since 2016 🚀",
+  "avatar": "https://github.com/mrabro.png",
+  "tags": ["Open Source", "wordpress", "javascript", "nodejs", "DevOps"],
+  "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/mrabro" },
+      { "platform": "github", "url": "https://github.com/mrabro" },
+      { "platform": "youtube", "url": "https://youtube.com/mrabro" }
```
Comment (kumarsonsoff3) [inline data/mrabro.json:10]: Please check & update your YouTube URL with the correct one. It seems to be broken! 🛠

image

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/mrabro.json (hunk 1 above)
Comment (mrabro) [reply data/mrabro.json:10]: oh, I forgot to replaced the platform and url: its actually facebook :)

**Review by kumarsonsoff3 — APPROVED** [review]
Awesome, Looks Good to me! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**